index.js

```diff
@@ -9,11 +9,11 @@ module.exports = {
   ],
   plugins: ['jest'],
   env: {
-    browser: true,
-    commonjs: true,
-    es6: true,
-    jest: true,
-    node: true,
+    'browser': true,
+    'commonjs': true,
+    'es6': true,
+    'jest': true,
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Typeform/eslint-config-typeform/pullRequest?prid=7367242)
```suggestion
    "jest": true,
```

Reply:
nooooooooooooo. we use single-quotes in Typeform! 😬